releases/release-1.22/release-notes/maps/pr-102873-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 102873
+releasenote:
+  text: Deprecate `--record` flag in `kubectl`. The `--record` flag is being replaced
+    with the [mechanism](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/859-kubectl-headers)
+    which annotates HTTP requests with kubectl command details.
```

Comment:
Should it be - which annotates HTTP requests with `kubectl` command details ?

Reply:
kubectl does not need to be quoted. ref: releases/release-1.21/release-notes/maps/pr-99833-map.yaml